common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -264,4 +264,9 @@ private ErrorStrings() {
      * Could not retrieve capabilities from broker apps.
      */
     public static final String FAILED_TO_GET_CAPABILITIES = "Could not get the capabilities";
+
+    /**
+     * Empty Context.
+     */
+    public static final String EMPTY_ANDROID_CONTEXT = "Empty Android context.";
```

Comment:
nit: The error message seems ambiguous, we should be explicit and say Activity not passed into BrokerTokenProvider

Reply:
done